### PR TITLE
Setup react boilerplate

### PR DESCRIPTION
**Pull request by musonant**
#### Description of Task to be completed?
- Setup babel
- Setup webpack
- Add demo react component

#### How should this be manually tested?
- Clone the repository
- On your terminal, start the server by running `npm run start`
- The server should start on open the page on the browser
- You should see a webpage titled "App Home"

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?(if applicable)
None

#### Screenshots
Screenshot 2019-09-12 at 2 44 27 PM

### CODE REVIEW

File: src/components/presentationals/Template.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+export default class Home extends Component {
```
Comment (seyi-adeleke) [inline src/components/presentationals/Template.jsx]: Lets rewrite this as a function. We should strive to use only functions and hooks in this project. We should only use a class if it'll be too unwieldy as a function

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './index.js',
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
```
Comment (seyi-adeleke) [inline webpack.config.js:8]: setup rules for scss and css

File: src/components/presentationals/Template.jsx (hunk 1 above)
Comment (seyi-adeleke) [reply src/components/presentationals/Template.jsx]: if you have any questions about using hooks let me know

Comment (musonant) [reply src/components/presentationals/Template.jsx]: I will rewrite this Component as a function then try out on the side how to use hooks. I'll reach out to you, if necessary. Thanks.

File: webpack.config.js (hunk 2 above)
Comment (musonant) [reply webpack.config.js:8]: True.

**Review by seyi-adeleke — APPROVED** [review]

**Pull request merged**